zest/releaser/git.py

```diff
@@ -61,85 +64,92 @@ def tag_url(self, version):
+        msg = u"Tagging {0}".format(version)
+        cmds = [[u'git', u'tag', version, u'-m', msg]]
+        if os.path.isdir(u'.git/svn'):
+            print(u"\nEXPERIMENTAL support for git-svn tagging!\n")
+            cur_branch = open(u'.git/HEAD').read().strip().split(u'/')[-1]
+            print(u"You are on branch {0}.".format(cur_branch))
+            if cur_branch != u'master':
+                print(u"Only the master branch is supported for "
+                      u"git-svn tagging.")
+                print(u"Please tag yourself.")
+                print(u"'git tag' needs to list tag named {0}.".format(version))
                 sys.exit(1)
-            cmd = [cmd]
 
             trunk = None
-            # In Git v2.0, the default prefix will change from "" (no prefix) to "origin/",
-            # try both here.
-            for t in ['.git/refs/remotes/trunk', '.git/refs/remotes/origin/trunk']:
+            # In Git v2.0, the default prefix will change from "" (no prefix)
+            # to "origin/", try both here.
+            for t in [u'.git/refs/remotes/trunk', u'.git/refs/remotes/origin/trunk']:
                 if os.path.isfile(t):
                     trunk = open(t).read()
 
             if not trunk:
-                print('No SVN remote found (only the default svn ' +
-                      'prefixes ("" or "origin/") are supported).')
+                print(u'No SVN remote found (only the default svn ' +
+                      u'prefixes ("" or "origin/") are supported).')
                 sys.exit(1)
 
-            local_head = open('.git/refs/heads/master').read()
+            local_head = open(u'.git/refs/heads/master').read()
             if local_head != trunk:
-                print("Your local master diverges from trunk.\n")
+                print(u"Your local master diverges from trunk.\n")
                 # dcommit before local tagging
-                cmd.insert(0, 'git svn dcommit')
+                cmds.insert(0, [u'git', u'svn', u'dcommit'])
             # create tag in svn
-            cmd.append('git svn tag -m "%s" %s' % (msg, version))
-        return cmd
+            cmds.append([u'git', u'svn', u'tag', u'-m', msg, version])
+        return cmds
 
     def cmd_checkout_from_tag(self, version, checkout_dir):
         if not (os.path.realpath(os.getcwd()) ==
                 os.path.realpath(checkout_dir)):
             # Specific to git: we need to be in that directory for the command
             # to work.
-            logger.warn("We haven't been chdir'ed to %s", checkout_dir)
+            logger.warn(u"We haven't been chdir'ed to {0}".format(checkout_dir))
             sys.exit(1)
-        return 'git checkout %s && git submodule update --init --recursive' % \
-            version
+        return [
+            [u'git', u'checkout', version],
+            [u'git', u'submodule', u'update', u'--init', u'--recursive']
+            ]
```

Comment:
I believe pep8-the-tool complains unless you align the closing `]` with `return`.  I also prefer that alignment, and also a trailing comma after the last item in a multi-line list, if the closing bracket is on a different line.